configure.ac

```diff
@@ -159,7 +159,7 @@ AC_CHECK_SIZEOF(bfd_vma, [], [#include <bfd.h>])
 # Checks for library functions.
 AC_FUNC_MALLOC
 AC_FUNC_MEMCMP
-AC_CHECK_FUNCS([bzero memset strstr strtoul])
+AC_CHECK_FUNCS([bzero memset strstr strtoul malloc])
```

Comment:
The bug went away. Probably this is related to a fix I applied to the ocaml compiler of my distribution. I only leave the updated stp patch here.